src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -599,9 +600,14 @@ function New-AzConnectedKubernetes {
 
         $PSBoundParameters.Add('ArcAgentryConfiguration', $arcAgentryConfigs)
 
-        Write-Verbose "Creating 'Kubernetes - Azure Arc' object in Azure"
+        Write-Host "Creating 'Kubernetes - Azure Arc' object in Azure"
```

Comment:
I believe `Write-Host` is deprecated in favour of `Write-Output`.  I think if you use `Invoke-ScriptAnalyser`, it will flag this.

Reply:
Sigh, `Write-Output` won't serve our needs to show up information in long wait process according to the doc https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-output?view=powershell-7.4#description
>  If Write-Output is the last command in the pipeline, the objects are displayed in the console.